### PR TITLE
Clean up TravisCI file

### LINKED ISSUE #5625: travis issue
https://github.com/ember-cli/ember-cli/commit/ac46b8da953b6d2259b6ae0c3508c6daa888b1eb

### PR DESCRIPTION
**Pull request by Turbo87**
resolves #5625

### CODE REVIEW

**Comment by Turbo87** [comment]
@stefanpenner seems to work fine

File: .travis.yml [hunk 1]
```diff
@@ -12,18 +8,15 @@ node_js:
...
+    $HOME/.npm
+    $HOME/.cache # includes bowers cache
 
 before_install:
-  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
-  - test $TRAVIS_OS_NAME = "osx" && brew update && brew install phantomjs || test $TRAVIS_OS_NAME = "linux"
   - npm config set spin false
 
 install:
-  - node --version
```
Comment (stefanpenner) [inline .travis.yml]: Why are these removed?

Comment (Turbo87) [inline .travis.yml]: because TravisCI is showing them automatically already (see e.g. https://travis-ci.org/ember-cli/ember-cli/jobs/116771208#L151-L156)

**Comment by stefanpenner** [comment]
Perfect: :ship: 

It seems like Travis has been catching up to us!

**Comment by Turbo87** [comment]
@homu r+

**Comment by perlun** [comment]
Great @Turbo87 and @stefanpenner that you managed to sort it out! :sunny:
